### ISSUE TITLE
Outdated versions in INSTALL.md cause errors

### ISSUE THREAD

**Issue by nlaratta** [openstreetmap/openstreetmap-website#2909]
When following the instructions listed in INSTALL.md, which uses Ubuntu 18.04 and Ruby 2.5, we get these errors:

app/views/browse/note.html.erb:41: syntax error, unexpected ']'
app/views/browse/note.html.erb:41: syntax error, unexpected ']'
app/controllers/application_controller.rb:240:in 'web_timeout'
config/initializers/compressed_requests.rb:27:in 'call'
config/initializers/cors.rb:9:in 'call'

Further explanation from a team member: "All the errors I get are in relation to this app/views/browse/note.html.erb line 41 syntax error unexpected ']'.  The brackets have a [1...] inside, if I edit the note.html.erb file so that the brackets on line 41 have [1..2] within them, the error goes away."

When following the instructions using Ubuntu 20.04 with Ruby 2.7, we get no errors. We asked in IRC-dev if this means that INSTALL.md might be due for an update, or if we messed up. We were told by a few people that Ubuntu 20.04 LTS is the version used by the OSMF Servers. This goes against what's stated in the document.

Is INSTALL.md due for an update?

**Comment by OmarBitar** [comment]
so I did an installation with Ubuntu 20.04 and ruby 2.5 and got the same errors from this old closed issue: https://github.com/openstreetmap/openstreetmap-website/issues/2212. 
Localhost successfully renders the webpage and functions as expected, but I do get the errors mentioned earlier. I also added an image below for further documentation. I upgraded to 2.7, I can confirm that I get no errors. 

![Capture](https://user-images.githubusercontent.com/28638350/97203736-d9a74880-177a-11eb-8ba6-aae670e7a469.PNG)
